index.js

```diff
@@ -2463,7 +2463,28 @@ export default function(value, nominatim_object, optional_conf_parm) {
                      * use it and ignore lesser specific ones (for the
                      * country)
                      */
-                    return holiday_definitions[location_cc][location_state][type_of_holidays];
+                    var holidays_by_name = {};
+                    var country_holidays = holiday_definitions[location_cc][type_of_holidays];
+                    var state_holidays = holiday_definitions[location_cc][location_state][type_of_holidays];
+                    if (type_of_holidays === 'PH') {
+                        return holiday_definitions[location_cc][location_state][type_of_holidays];
```

Comment:
I guess you can comment the code after this return out for now to make it clear that for PH this is not effective currently.